core/services/relay/evm/llo/config_poller.go

```diff
@@ -70,12 +77,28 @@ func NewConfigPoller(lggr logger.Logger, lp LogPoller, cc ConfigCache, addr comm
 }
 
 func newConfigPoller(lggr logger.Logger, lp LogPoller, cc ConfigCache, addr common.Address, donID uint32, instanceType InstanceType, fromBlock uint64) *configPoller {
+	donIDTopic := DonIDToBytes32(donID)
+	exprs := []query.Expression{
+		logpoller.NewAddressFilter(addr),
+		query.Or(
+			logpoller.NewEventSigFilter(ProductionConfigSet),
```

Comment:
sorry, is it really hardcoded in the code that the types can be  ProductionConfigSet and StagingConfigSet? If yes, how production (as type) is different from the staging?

Reply:
Yes it is hardcoded. Events are defined here: https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/llo-feeds/v0.5.0/configuration/interfaces/IConfigurator.sol